src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -101,6 +102,8 @@ protected void install() throws InterruptedException {
 
         context.get(UseCases.class).init(context);
         context.get(CertificateManager.class).updateCertificatesConfiguration(new CertificatesConfig(getConfig()));
+        context.get(ConnectivityInfoCache.class)
```

Comment:
look to how we've done the other config based things, we do have stuff that interacts with the runtime config and we should follow that.

Reply:
addressed